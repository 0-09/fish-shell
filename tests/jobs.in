jobs -q; echo $status
sleep 5 &
sleep 5 &
jobs -c
jobs -q; echo $status
bg -23 1 2>/dev/null
or echo bg: invalid option -23 >&2
fg 3
bg 3
sleep 1 &
disown
jobs -c
disown foo
disown (jobs -p)
or exit 0

# https://github.com/fish-shell/fish-shell/issues/5824
function foo
    sleep 1 &
	jobs -c
end
foo
