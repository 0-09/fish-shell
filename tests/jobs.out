1
Command
sleep
sleep
0
Command
sleep
sleep
Command
sleep
sleep is done
